API_Auto/src/test/java/base/CrudOperation.java

```diff
@@ -0,0 +1,22 @@
+package base;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+import static base.FinalServiceUrl.*;
+
+public class CrudOperation extends EndPointServiceUrl {
```

Comment:
No need to extend EndPointServiceUrl class if not reusing the base class methods

Reply:
done